### PR TITLE
webfaf: Repair "About" page

### LINKED ISSUE #766: About page not working -  Page not found.
https://retrace.fedoraproject.org/faf/about

About page stopped working after https://github.com/abrt/faf/commit/81cef782e7563e1a0d26ff36840afe2c9765e51c

### PR DESCRIPTION
**Pull request by mgrabovsky**
Since there are no more RST pages, switched dependency on Flask-RSTPages
for markdown2, which renders Markdown documents to HTML.

Closes #766

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

File: src/webfaf/webfaf_main.py [hunk 1]
```diff
@@ -127,8 +127,10 @@ def root():
 @app.route('/about')
 @cache(hours=24)
 def about():
-    rstdoc = RSTPages().get("about")
-    return flask.render_template("rstpage.html", rstdoc=rstdoc)
+    path = flask.safe_join(app.config['TEMPLATES_DIR'], "about.md")
+    html = markdown2.markdown_path(path)
+    mddoc = {"body": html, "title": "About FAF"}
```
Comment (mkutlak) [inline src/webfaf/webfaf_main.py]: s/About FAF/About ABRT Analytics/

See https://github.com/abrt/faf/issues/746

File: faf.spec.in [hunk 2]
```diff
@@ -871,7 +871,7 @@ fi
 %{python3_sitelib}/webfaf/templates/500.html
 %{python3_sitelib}/webfaf/templates/about.md
 %{python3_sitelib}/webfaf/templates/base.html
-%{python3_sitelib}/webfaf/templates/rstpage.html
```
Comment (mkutlak) [inline faf.spec.in]: Delete the `rstpage.html` file from sources.

**Comment by mkutlak** [comment]
Othetwise LGTM :+1:

**Comment by mgrabovsky** [comment]
@mkutlak Updated.

**Comment by mkutlak** [comment]
Thank you

**Pull request merged**